### ISSUE TITLE
PlatformException device_not_capable

### ISSUE THREAD

**Issue by jsnydersss** [fluttercommunity/flutter_sms#34]
Hello,
I put in my AndroidManifest.xml file:

<uses-permission android:name="android.permissions.SEND_SMS"/>

and continuously get on my physical Pixel 5

PlatformException(device_not_capable, The current device is not capable of sending text messages., A device may be unable to send messages if it does not support messaging or if it is not currently configured to send messages. This only applies to the ability to send text messages via iMessage, SMS, and MMS., null)

My app requires 3 permissions: location, phone and sms. The first two the permission is granted. However for SMS I cannot request the permission and constantly returns undetermined.

My Pixel 5 can send/receive texts as normally used.

**Comment by rodydavis** [comment]
fixed on version `2.3.1` reopen if it is still an issue

**Comment by makis73** [comment]
Hello, 
About this problem about  PlatformException(device_not_capable, The current device is not capable ...
for me happens only when running on Android 11 even with flutter_sms 2.3.1
The solution i found is:
I needed to add this to my manifest, just above <application tab:

 ```
 <queries>
        <intent>
            <action android:name="android.intent.action.SENDTO" />
            <data android:scheme="smsto"/>
        </intent>
 </queries>
```



more infos here [https://stackoverflow.com/questions/64647782/intent-for-sending-text-messages-no-longer-working-on-android-11](url)

and then follow this answer : [https://stackoverflow.com/questions/64502943/exception-thrown-in-build-by-image-picker](url)
Hope this will help ...
